aten/src/ATen/quantized/Quantizer.h

```diff
@@ -171,13 +161,13 @@ struct TORCH_API PerChannelAffineQuantizer : public AffineQuantizer {
  * be exactly represented in the quantized space. We can get additional precision by
  * using floating point values for zero point.
  */
-struct TORCH_API PerChannelAffineFloatQParamsQuantizer : public PerChannelAffineQuantizer {
-  explicit PerChannelAffineFloatQParamsQuantizer(
+struct TORCH_API PerChannelFloatQParamsQuantizer : public PerChannelQuantizer {
+  explicit PerChannelFloatQParamsQuantizer(
       ScalarType scalar_type,
       Tensor scales,
       Tensor zero_points,
       int64_t axis)
-      : PerChannelAffineQuantizer(scalar_type,
+      : PerChannelQuantizer(scalar_type,
```

Comment:
is there a symmetric/affine variant for this quantizer?

Reply:
No, this one only has affine variant.